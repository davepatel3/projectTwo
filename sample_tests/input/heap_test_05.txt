6505
create_file_root
64
SUPToL
get_file_size_path
/SUPToL
print_layout
rename
1
JLpKcknjUwxiZ5kZL
create_directory_root
9pl-n-nHQHSjaaoUUa
create_file_root
13
g1dSnZ5UuUYGHeg9pOJi
create_directory_root
GUUoDPjHL4Yz65V
create_link
4
ADgQcVOZfUwWShkl9I
2
create_file
31
TvZI6EQ
4
create_file_root
239
Mh8ep-8
create_link
6
h84It4016
2
get_file_size_path
/9pl-n-nHQHSjaaoUUa
get_name
5
create_file
76
cJ0z4Hef_2LGXQCP
4
get_largest_file_handle
create_directory
GBS91Sb
0
create_directory
5wJubSg
4
create_directory
EzhPiQC
4
get_file_size
1
get_name
11
print_layout
get_largest_file_handle
rename
10
Fnfq6A
create_file
233
BpZQqekCIiDn5
4
exist
10
rename
9
AOhMuDnsHYE
create_directory
hW-0ubvXGjRAiG
4
create_file_root
114
RjhQosocTG8L qLE
get_file_size
9
get_name
8
print_layout
get_absolute_path
6
get_available_size
exist
13
create_file
66
m2D-PvxaQzNzZtU
2
get_available_size
create_link
10
PULn6WmMRJT7qR5kP2
2
create_link
8
aPhXpcXb NGLpPFGiM
4
create_directory
DDczXdTXUP0rH3le
2
get_available_size
create_file_root
99
-N3aQOBpOPOqbcZPpu
get_handle
/JLpKcknjUwxiZ5kZL
create_link
10
vz272yE2dIX5Dwkbi
14
create_directory
MaY03AFq
2
follow
18
create_file
4
U14xDX2URV4M
14
remove
1
remove
4
create_file
81
udER9Kj QucdX
4
get_file_size_path
/9pl-n-nHQHSjaaoUUa
get_file_size_path
/GUUoDPjHL4Yz65V/hW-0ubvXGjRAiG/vz272yE2dIX5Dwkbi
create_directory
gbhyij
10
create_directory
m6BvlUiMzq-2aCs7
4
create_file
19
XOGN9 U_2sw
25
create_file
122
N0cjS6P
0
get_largest_file_handle
remove
17
create_file
10
UAXACwjsO_V9Kv7sa
11
exist
8
get_largest_file_handle
get_file_size_path
/GUUoDPjHL4Yz65V/aPhXpcXb NGLpPFGiM
rename
17
D3Vk7sDil2I7rxaOmC
get_absolute_path
27
get_largest_file_handle
create_file
63
FiKtB5alpC PDz
11
get_file_size
6
print_layout
print_layout
create_file
8
-VYHbjXOuy
19
create_directory
s3k4-
4
create_file
107
bagunFfXEHE6dSZu
30
get_largest_file_handle
rename
8
f1LsK72
create_file
29
uzXNBEpi-Yc5
19
create_directory
hv7pwLDGHk
25
rename
12
tFoW5F
exist
16
print_layout
create_link
13
N7Pl_XFNuWefZP7Y
22
get_available_size
exist
28
create_file
58
4lluT2-uBG
11
get_absolute_path
34
exist
2
create_directory
Cf6VKm2AAB6
19
create_file
7
DVYvYK1E6K3QcysyK
10
get_absolute_path
23
exist
1
get_file_size_path
/GUUoDPjHL4Yz65V/s3k4-
get_absolute_path
34
create_link
27
AY5qEnEO_rCT7
4
get_handle
/GUUoDPjHL4Yz65V/aPhXpcXb NGLpPFGiM
create_link
31
Fx6bwjkBk
30
remove
16
get_absolute_path
38
create_file
5
tJw8C_SyU2_d-
33
create_link
14
VC7CgA0Y0eER2eye
22
create_file
16
cpnm7eNrhjX qHAg9yO
36
rename
20
tYLHJMJ4Lv54ePkZdgx
create_link
22
ksVvfsLGA4CdUm ZDKy6
12
remove
30
get_largest_file_handle
create_link
25
1BtVBPluJJVT4n
19
get_absolute_path
18
create_file
104
2v5 cEqdQ_kZY19ro
33
create_file
24
CUxwazXmXkq4djh 9
36
create_file
249
AktfkG
10
create_directory
s9nqdLV
4
create_file
189
iBIaMUGlRiF6qPtEYYJ
10
get_file_size_path
/9pl-n-nHQHSjaaoUUa/MaY03AFq/VC7CgA0Y0eER2eye
create_file
17
K 1 U6lRG jIJLpK 
25
get_available_size
get_handle
/GUUoDPjHL4Yz65V/aPhXpcXb NGLpPFGiM
get_available_size
create_file
9
j9PmZQU
10
get_file_size
16
print_layout
create_link
42
e2WdiTGBnmPyd4Ul
33
create_directory
HviR6dhTTEL
24
create_directory
OFHUTuPatA52Vs
4
get_name
2
create_link
20
i8QAT1JRd
25
get_name
29
quit
