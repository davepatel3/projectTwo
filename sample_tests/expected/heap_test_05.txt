create_file_root
1
get_file_size_path
64
print_layout
[F]SUPToL (size = 64)

rename
create_directory_root
2
create_file_root
3
create_directory_root
4
create_link
5
create_file
6
create_file_root
7
create_link
8
get_file_size_path
Invalid handle!
get_name
ADgQcVOZfUwWShkl9I
create_file
9
get_largest_file_handle
7
create_directory
10
create_directory
11
create_directory
12
get_file_size
64
get_name
5wJubSg
print_layout
[F]JLpKcknjUwxiZ5kZL (size = 64)
[D]9pl-n-nHQHSjaaoUUa
	[L]ADgQcVOZfUwWShkl9I [->/GUUoDPjHL4Yz65V]
	[L]h84It4016 [->/GUUoDPjHL4Yz65V/TvZI6EQ]
[F]g1dSnZ5UuUYGHeg9pOJi (size = 13)
[D]GUUoDPjHL4Yz65V
	[F]TvZI6EQ (size = 31)
	[F]cJ0z4Hef_2LGXQCP (size = 76)
	[D]5wJubSg
	[D]EzhPiQC
[F]Mh8ep-8 (size = 239)
[D]GBS91Sb

get_largest_file_handle
7
rename
create_file
13
exist
1
rename
create_directory
14
create_file_root
15
get_file_size
76
get_name
h84It4016
print_layout
[F]JLpKcknjUwxiZ5kZL (size = 64)
[D]9pl-n-nHQHSjaaoUUa
	[L]ADgQcVOZfUwWShkl9I [->/GUUoDPjHL4Yz65V]
	[L]h84It4016 [->/GUUoDPjHL4Yz65V/TvZI6EQ]
[F]g1dSnZ5UuUYGHeg9pOJi (size = 13)
[D]GUUoDPjHL4Yz65V
	[F]TvZI6EQ (size = 31)
	[F]AOhMuDnsHYE (size = 76)
	[D]5wJubSg
	[D]EzhPiQC
	[F]BpZQqekCIiDn5 (size = 233)
	[D]hW-0ubvXGjRAiG
[F]Mh8ep-8 (size = 239)
[D]Fnfq6A
[F]RjhQosocTG8L qLE (size = 114)

get_absolute_path
/GUUoDPjHL4Yz65V/TvZI6EQ
get_available_size
5735
exist
1
create_file
16
get_available_size
5669
create_link
17
create_link
18
create_directory
19
get_available_size
5669
create_file_root
20
get_handle
1
create_link
21
create_directory
22
follow
6
create_file
23
remove
remove
create_file
1
get_file_size_path
Invalid handle!
get_file_size_path
Invalid handle!
create_directory
24
create_directory
25
create_file
26
create_file
27
get_largest_file_handle
7
remove
create_file
17
exist
1
get_largest_file_handle
7
get_file_size_path
31
rename
get_absolute_path
/N0cjS6P
get_largest_file_handle
7
create_file
28
get_file_size
31
print_layout
[D]9pl-n-nHQHSjaaoUUa
	[L]ADgQcVOZfUwWShkl9I [->/GUUoDPjHL4Yz65V]
	[L]h84It4016 [->/GUUoDPjHL4Yz65V/TvZI6EQ]
	[F]m2D-PvxaQzNzZtU (size = 66)
	[D]DDczXdTXUP0rH3le
	[D]MaY03AFq
[F]g1dSnZ5UuUYGHeg9pOJi (size = 13)
[D]GUUoDPjHL4Yz65V
	[F]TvZI6EQ (size = 31)
	[F]AOhMuDnsHYE (size = 76)
	[D]5wJubSg
		[F]D3Vk7sDil2I7rxaOmC (size = 10)
		[F]FiKtB5alpC PDz (size = 63)
	[D]EzhPiQC
	[F]BpZQqekCIiDn5 (size = 233)
	[D]hW-0ubvXGjRAiG
		[L]vz272yE2dIX5Dwkbi [->/Fnfq6A]
		[F]U14xDX2URV4M (size = 4)
	[L]aPhXpcXb NGLpPFGiM [->/GUUoDPjHL4Yz65V/TvZI6EQ]
	[F]udER9Kj QucdX (size = 81)
	[D]m6BvlUiMzq-2aCs7
		[F]XOGN9 U_2sw (size = 19)
[F]Mh8ep-8 (size = 239)
[D]Fnfq6A
	[D]gbhyij
[F]RjhQosocTG8L qLE (size = 114)
[F]-N3aQOBpOPOqbcZPpu (size = 99)
[F]N0cjS6P (size = 122)

print_layout
[D]9pl-n-nHQHSjaaoUUa
	[L]ADgQcVOZfUwWShkl9I [->/GUUoDPjHL4Yz65V]
	[L]h84It4016 [->/GUUoDPjHL4Yz65V/TvZI6EQ]
	[F]m2D-PvxaQzNzZtU (size = 66)
	[D]DDczXdTXUP0rH3le
	[D]MaY03AFq
[F]g1dSnZ5UuUYGHeg9pOJi (size = 13)
[D]GUUoDPjHL4Yz65V
	[F]TvZI6EQ (size = 31)
	[F]AOhMuDnsHYE (size = 76)
	[D]5wJubSg
		[F]D3Vk7sDil2I7rxaOmC (size = 10)
		[F]FiKtB5alpC PDz (size = 63)
	[D]EzhPiQC
	[F]BpZQqekCIiDn5 (size = 233)
	[D]hW-0ubvXGjRAiG
		[L]vz272yE2dIX5Dwkbi [->/Fnfq6A]
		[F]U14xDX2URV4M (size = 4)
	[L]aPhXpcXb NGLpPFGiM [->/GUUoDPjHL4Yz65V/TvZI6EQ]
	[F]udER9Kj QucdX (size = 81)
	[D]m6BvlUiMzq-2aCs7
		[F]XOGN9 U_2sw (size = 19)
[F]Mh8ep-8 (size = 239)
[D]Fnfq6A
	[D]gbhyij
[F]RjhQosocTG8L qLE (size = 114)
[F]-N3aQOBpOPOqbcZPpu (size = 99)
[F]N0cjS6P (size = 122)

create_file
29
create_directory
30
create_file
31
get_largest_file_handle
7
rename
create_file
32
create_directory
33
rename
exist
1
print_layout
[D]9pl-n-nHQHSjaaoUUa
	[L]ADgQcVOZfUwWShkl9I [->/GUUoDPjHL4Yz65V]
	[L]f1LsK72 [->/GUUoDPjHL4Yz65V/TvZI6EQ]
	[F]m2D-PvxaQzNzZtU (size = 66)
	[D]DDczXdTXUP0rH3le
		[F]-VYHbjXOuy (size = 8)
		[F]uzXNBEpi-Yc5 (size = 29)
	[D]MaY03AFq
[F]g1dSnZ5UuUYGHeg9pOJi (size = 13)
[D]GUUoDPjHL4Yz65V
	[F]TvZI6EQ (size = 31)
	[F]AOhMuDnsHYE (size = 76)
	[D]5wJubSg
		[F]D3Vk7sDil2I7rxaOmC (size = 10)
		[F]FiKtB5alpC PDz (size = 63)
	[D]tFoW5F
	[F]BpZQqekCIiDn5 (size = 233)
	[D]hW-0ubvXGjRAiG
		[L]vz272yE2dIX5Dwkbi [->/Fnfq6A]
		[F]U14xDX2URV4M (size = 4)
	[L]aPhXpcXb NGLpPFGiM [->/GUUoDPjHL4Yz65V/TvZI6EQ]
	[F]udER9Kj QucdX (size = 81)
	[D]m6BvlUiMzq-2aCs7
		[F]XOGN9 U_2sw (size = 19)
		[D]hv7pwLDGHk
	[D]s3k4-
		[F]bagunFfXEHE6dSZu (size = 107)
[F]Mh8ep-8 (size = 239)
[D]Fnfq6A
	[D]gbhyij
[F]RjhQosocTG8L qLE (size = 114)
[F]-N3aQOBpOPOqbcZPpu (size = 99)
[F]N0cjS6P (size = 122)

create_link
34
get_available_size
5191
exist
1
create_file
35
get_absolute_path
/9pl-n-nHQHSjaaoUUa/MaY03AFq/N7Pl_XFNuWefZP7Y
exist
1
create_directory
36
create_file
37
get_absolute_path
/GUUoDPjHL4Yz65V/hW-0ubvXGjRAiG/U14xDX2URV4M
exist
1
get_file_size_path
Invalid handle!
get_absolute_path
/9pl-n-nHQHSjaaoUUa/MaY03AFq/N7Pl_XFNuWefZP7Y
create_link
38
get_handle
18
create_link
39
remove
get_absolute_path
/GUUoDPjHL4Yz65V/AY5qEnEO_rCT7
create_file
16
create_link
40
create_file
41
rename
create_link
42
remove
get_largest_file_handle
7
create_link
43
get_absolute_path
/GUUoDPjHL4Yz65V/aPhXpcXb NGLpPFGiM
create_file
44
create_file
45
create_file
46
create_directory
47
create_file
48
get_file_size_path
Invalid handle!
create_file
49
get_available_size
4588
get_handle
18
get_available_size
4588
create_file
50
get_file_size
5
print_layout
[D]9pl-n-nHQHSjaaoUUa
	[L]ADgQcVOZfUwWShkl9I [->/GUUoDPjHL4Yz65V]
	[L]f1LsK72 [->/GUUoDPjHL4Yz65V/TvZI6EQ]
	[D]DDczXdTXUP0rH3le
		[F]-VYHbjXOuy (size = 8)
		[F]uzXNBEpi-Yc5 (size = 29)
		[D]Cf6VKm2AAB6
			[F]cpnm7eNrhjX qHAg9yO (size = 16)
			[F]CUxwazXmXkq4djh 9 (size = 24)
		[L]1BtVBPluJJVT4n [->/GUUoDPjHL4Yz65V/m6BvlUiMzq-2aCs7]
	[D]MaY03AFq
		[L]N7Pl_XFNuWefZP7Y [->/GUUoDPjHL4Yz65V/BpZQqekCIiDn5]
		[L]VC7CgA0Y0eER2eye [->/GUUoDPjHL4Yz65V/hW-0ubvXGjRAiG]
[F]g1dSnZ5UuUYGHeg9pOJi (size = 13)
[D]GUUoDPjHL4Yz65V
	[F]TvZI6EQ (size = 31)
	[F]AOhMuDnsHYE (size = 76)
	[D]5wJubSg
		[F]D3Vk7sDil2I7rxaOmC (size = 10)
		[F]FiKtB5alpC PDz (size = 63)
		[F]4lluT2-uBG (size = 58)
	[D]tFoW5F
		[L]ksVvfsLGA4CdUm ZDKy6 [->/9pl-n-nHQHSjaaoUUa/MaY03AFq]
	[F]BpZQqekCIiDn5 (size = 233)
	[D]hW-0ubvXGjRAiG
		[L]vz272yE2dIX5Dwkbi [->/Fnfq6A]
		[F]U14xDX2URV4M (size = 4)
	[L]aPhXpcXb NGLpPFGiM [->/GUUoDPjHL4Yz65V/TvZI6EQ]
	[F]udER9Kj QucdX (size = 81)
	[D]m6BvlUiMzq-2aCs7
		[F]XOGN9 U_2sw (size = 19)
		[D]hv7pwLDGHk
			[F]tJw8C_SyU2_d- (size = 5)
			[F]2v5 cEqdQ_kZY19ro (size = 104)
		[F]K 1 U6lRG jIJLpK  (size = 17)
	[D]s3k4-
		[F]bagunFfXEHE6dSZu (size = 107)
		[L]Fx6bwjkBk [->/GUUoDPjHL4Yz65V/s3k4-/bagunFfXEHE6dSZu]
	[L]AY5qEnEO_rCT7 [->/N0cjS6P]
	[D]s9nqdLV
[F]Mh8ep-8 (size = 239)
[D]Fnfq6A
	[D]gbhyij
	[F]DVYvYK1E6K3QcysyK (size = 7)
	[F]AktfkG (size = 249)
	[F]iBIaMUGlRiF6qPtEYYJ (size = 189)
	[F]j9PmZQU (size = 9)
[F]RjhQosocTG8L qLE (size = 114)
[F]tYLHJMJ4Lv54ePkZdgx (size = 99)
[F]N0cjS6P (size = 122)

create_link
51
create_directory
52
create_directory
53
get_name
9pl-n-nHQHSjaaoUUa
create_link
54
get_name
-VYHbjXOuy
quit
